app/posts/record-a-vaccination/2025/03/notifying-users-about-updates.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Notifying users about updates"
+date: 2025-03-25
+---
+We wanted to explore how we might tell RAVs users about any updates to the service which might impact them. This could include new features to RAVs such as asking them to complete new fields when adding a vaccine batch or larger scale changes such as a new streamlined journey of how users can record vaccinations in RAVs.
+Both new feature examples are on the release roadmap for RAVs, so we were able to use these scenarios to demonstrate with users and give the designs appropriate scenarios for which alerts or notifications might be used.
+
+## Desk research
+
+As there are no current notifications in the NHS Design system, we reached out to the internal design community at NHS England to see if there was any precedents elsewhere as well as doing some initial research in to what other government departments might have done to tackle this problem.
+
+The following list was checked as well as searches online to see what components worked best and in what context:
+
+https://github.com/ctdesign/gov-design-systems-list
```

Comment:
This link isn’t currently clickable. Could make it a link within the previous sentence? Something like "A [list of government design systems] was checked..."?